platform/platform-impl/src/com/intellij/openapi/wm/impl/FrameInfoHelper.kt

```diff
@@ -77,7 +78,7 @@ private fun updateFrameInfo(frameHelper: ProjectFrameHelper, lastNormalFrameBoun
   val frame = frameHelper.frame
   var extendedState = frame.extendedState
   if (SystemInfo.isMacOSLion) {
-    val peer = AWTAccessor.getComponentAccessor().getPeer(frame)
+    val peer = AWTAccessor.getComponentAccessor().getPeer(frame) as ComponentPeer?
```

Comment:
Why is this cast necessary if we do that the `is` check below?

Reply:
The commit message explains:
> In JDK 8 `AWTAccessor.getPeer` returns raw `ComponentPeer`
> but in JDK 11 it returns `<T extends ComponentPeer>`.

Without the cast Kotlin compilation fails with `Not enough information to infer type variable T`.